### ISSUE TITLE
No file paths defined

### ISSUE THREAD

**Issue by InfoAtOnTime** [download-monitor/download-monitor#280]
Hello, I've tried this a number of ways but I continue to get the same issue, somehow the connection to the file is lost. After filling out the form, an incorrect message shows up as the link "Download #204 File Version" which the download # is correct, but when this plugin worked, it displayed the name of the file. This was my first indicator that something wasn't quite right, tried clicking download anyway and that's when I got the No File Paths Defined error. I've done and redone the procedure using different PDFs, different forms, different pages, etc. - always this same error.  Can you help? Thanks!

**Comment by barrykooij** [comment]
Does the file works if you go to the direct URL you enter in your version URL field?

**Comment by InfoAtOnTime** [comment]
Hi Barry, I ended up figuring out what was going on, I was using the wrong ID number. When setting up the download item, in the "Edit Download" section, there is a number listed in the "Downloadable Files/Versions" - that is the number I was using. Once I updated the page and went back to the list of downloads, I noticed there was a different number next to the download name - apparently that is the one to use. The numbers weren't that far apart so I didn't notice at first that it was different.
